molecule.yml

```diff
@@ -50,18 +53,18 @@ vagrant:
     - name: nginx.vm
       ansible_groups:
         - group01
-        - nginx
       interfaces:
         - network_name: private_network
-          type: dhcp
+          type: static
```

Comment:
Just for curiosity, why is this needed?

Reply:
This is because the vagrant-hostmanager plugin doesn't manage host in dhcp mode :(